wls-gui-wahllokalsystem/src/composables/wahlvorbereitung/wahlvorbereitungMapper.ts

```diff
@@ -72,6 +75,33 @@ export function useWahlvorbereitungMapper() {
     };
   }
 
+  function toBriefwahlvorbereitungModel(
+    briefwahlvorbereitungDTO: BriefwahlvorbereitungDTO
+  ): Wahlvorbereitung {
+    const urnenAnzahlModel =
+      briefwahlvorbereitungDTO.urnenAnzahl?.map((wahlurneDTO) =>
+        _toWahlurneModel(wahlurneDTO)
+      ) ?? [];
+    return {
+      wahlbezirkID: briefwahlvorbereitungDTO.wahlbezirkID,
+      urneVersiegelt:
+        briefwahlvorbereitungDTO.urnenAnzahl[0]?.urneVersiegelt ?? false,
+      urnenAnzahl: urnenAnzahlModel,
+    };
+  }
+
+  function toBriefwahlvorbereitungWriteDto(
+    briefwahlvorbereitung: Wahlvorbereitung
+  ): BriefwahlvorbereitungWriteDTO {
+    const urnenAnzahlDto =
+      briefwahlvorbereitung.urnenAnzahl?.map((wahlurneDTO) =>
+        _toWahlurneDto(wahlurneDTO, briefwahlvorbereitung.urneVersiegelt)
+      ) ?? [];
```

Comment:
`urnenAnzahl` ist nicht optional.

```suggestion
    const urnenAnzahlDto = briefwahlvorbereitung.urnenAnzahl.map(
      (wahlurneDTO) =>
        _toWahlurneDto(wahlurneDTO, briefwahlvorbereitung.urneVersiegelt)
    );
```